packages/playground/worker/index.html

```diff
@@ -20,18 +20,22 @@
   <span class="tick-count">0</span>
 </div>
 
-<p>new Worker(new Url('path', import.meta.url))</p>
+<p>new Worker(new Url('path', import.meta.url), { type: 'module' })</p>
 <div class="worker-import-meta-url"></div>
 
-<p>new SharedWorker(new Url('path', import.meta.url))</p>
+<p>new SharedWorker(new Url('path', import.meta.url), { type: 'module' })</p>
 <div class="shared-worker-import-meta-url"></div>
 
+<p>new Worker(new Url('path', import.meta.url))</p>
```

Comment:
Could we add another test case for `{ type: 'module' }`?

Reply:
any else case? `{ type: 'module' }` test above this line.

Reply:
🤦🏼 sorry, I'll try again.

Could we add another test case for an explicit `{ type: 'classic' }`?